src/components/DrawingBoard.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { MouseEvent, useEffect, useRef } from 'react';
+import './DrawingBoard.css';
+
+export interface DrawingBoardProps {
+    imageUrl: string;
+    index: number;
+}
+
+enum drawingState {
+    NotStarted,
+    StartPoint,
+    Completed,
+}
+
+interface DrawingLine {
+    x1: number,
+    y1: number,
+    x2: number,
+    y2: number,
+}
+
+
+const DrawingBoard: React.FC<DrawingBoardProps> = (props: DrawingBoardProps) => {
+
+    let history: DrawingLine[] = [];
```

Comment:
the name history is misleading since it represents the drawings which were made so far.
This is part of the component state as well